specs/deneb/polynomial-commitments.md

```diff
@@ -429,14 +429,15 @@ def verify_kzg_proof_batch(commitments: Sequence[KZGCommitment],
 #### `compute_kzg_proof`
 
 ```python
-def compute_kzg_proof(blob: Blob, z: Bytes32) -> KZGProof:
+def compute_kzg_proof(blob: Blob, z: Bytes32) -> Tuple[KZGProof, Bytes32]:
```

Comment:
I think it would follow the convention to return a `BLSFieldElement` here since it's a validated/normalized value returned by the library (like `KZGProof`).

That said, this would be the first public method that returns a `BLSFieldElement` so it would need a bit more setup on the library/bindings side.

Reply:
Ugh scratch that. I thought that `BLSFieldElement` is also Bytes32 but it's actually a BLS library object.